File: 백준/Bronze/1934. 최소공배수/최소공배수.cc
#include <stdio.h>
#include <iostream>

using namespace std;

int gcd(int a, int b) {
    while(b!=0) {
        int t = a;
        a = b;
        b = t % b;
    }
    return a;
}
int lcm(int a, int b) {
    return a*b/(gcd(a, b));
}
int main(void) {
    int k;
    cin >> k;
    while(k--) {
     int n, m;
    scanf("%d %d", &n, &m);
    printf("%d\n", lcm(n, m));   
    }
}

File: 백준/Silver/13241. 최소공배수/최소공배수.py
def gcd(a, b):
    while b!=0:
        a, b = b, a%b
    return a
def lcm(a, b):
    return a*b//gcd(a, b)
def main():
    n, m = map(int, input().split())

    print(lcm(n, m))


if __name__ == "__main__":
    main()